Model/Transfer/CustomFields/Parts/Item.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace Punchout2Go\Punchout\Model\Transfer\CustomFields\Parts;
+
+use Magento\Quote\Api\Data\CartItemInterface;
+use Punchout2Go\Punchout\Model\Transfer\CustomFields\CartItemPartInterface;
+
+/**
+ * Class Item
+ * @package Punchout2Go\Punchout\Model\Transfer\CustomFields\Parts
+ */
+class Item implements CartItemPartInterface
+{
+    /**
+     * @param CartItemInterface $product
+     * @param string $path
+     * @return string
+     */
+    public function handle(CartItemInterface $product, string $path): string
+    {
+        $returnValue = '';
+        $productData = $product->getData($path);
+        if (isset($productData)) {
+           $returnValue = $product->getData($path);
+        }
+        return $returnValue;
```

Comment:
Seems like this could be simplified?
```
return $product->getData($path) ?? '';
```

Reply:
the value could be null or not isset, so this covers it.

Reply:
The Null Coalescing Operator looks for both.

Reply:
ok, updated